pages/index.vue

```diff
@@ -95,7 +112,7 @@
 
     head() {
       return {
-        title: this.$pageHeadTitle(this.name),
+        title: this.$t('homepageTitle') + this.$pageHeadTitle(),
```

Comment:
The $t('homepageTitle') seems missing in the language file.

Reply:
it shouldn't be coming from the translation file. It should be the one from 'this.name'

Reply:
ok fixed